git.lua

```diff
@@ -156,11 +156,17 @@ local function checkout_spec_generator(token)
     --     since it is not added automatically by readline (see previous point)
     clink.matches_are_files(0)
     clink.match_display_filter = function ()
+        local pre = ''
+        local suf = ''
+        if (clink.version_encoded or 0) >= 10010009 and rl.isvariabletrue('colored-stats') then
+            pre = '\x1b['..settings.get('color.git.star')..'m'
+            suf = '\x1b['..settings.get('color.filtered')..'m'
```

Comment:
There's already a coupple of functions in [`color.lua`](https://github.com/vladimir-kotikov/clink-completions/blob/master/modules/color.lua) to enable color output - could you please reuse those functions (possibly updating them to add extra functionality if needed) instead of doing manual string concatenations?

Reply:
Ok.

Reply:
Done.